src/components/code/SpecCode.vue

```diff
@@ -426,4 +426,8 @@ export default {
   pre {
     margin: 0;
   }
+
+  code {
+    min-height: 20em;
+  }
```

Comment:
I tried this before but it's a bit weird the have a code block with line number only a the beginning

Reply:
Fix in #51